docs/.vuepress/config.js

```diff
@@ -266,7 +266,8 @@ const sidebarContribute = [
       },
       ['/Contribute/Design', 'Design'],
       ['/Contribute/Translate', 'Translate'],
-      ['/Contribute/Misc', 'Miscellaneous']
+      ['/Contribute/Misc', 'Miscellaneous'],
+      ['/Contribute/Agenda', 'Calls Calender']
```

Comment:
It's calendar! 😄 
```suggestion
      ['/Contribute/Agenda', 'Calls Calendar']
```

Reply:
Should we call these office hours, open calls? Calls calendar doesn't really give context on what's going on?

Reply:
What ya say about "Open office hours" ?

Reply:
That may imply there's an office or even working hours :) How about "Open Community Calls"?

Reply:
Allright will do :)